samples/CGCatalog/CGCatalog/CGDrawView.h

```diff
@@ -0,0 +1,26 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#import <UIKit/UIKit.h>
+
+@interface CGDrawView : UIView
+- (id)initWithFrame:(CGRect)rect lineWidth:(CGFloat)width color:(CGColorRef)color;
+- (void)setDrawBlock:(void (^)(void))block;
+@property (assign) CGFloat lineWidth;
+@property (assign) CGColorRef lineColor;
```

Comment:
I don't think you can retain/copy a CGColorRef since it's not an object?

---
In reply to: [80593760](https://github.com/Microsoft/WinObjC/pull/1033#discussion_r80593760) [](ancestors = 80593760)